Tests/ProtobufferCodingTests/Encoding/OptionalEncodingTests.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  OptionalEncodingTests.swift
+//  
+//
+//  Created by Moritz Schüll on 19.01.21.
+//
+
+import Foundation
+import XCTest
+@testable import ProtobufferCoding
+
+class OptionalEncodingTests: XCTestCase {
+    func testEncodeSinglePositiveOptionalInt32() throws {
+        let expected = Data([8, 185, 96])
+        let number: Int32? = 12345
+
+        let encoded = try ProtobufferEncoder().encode(number)
+        XCTAssertEqual(encoded, expected, "testEncodeSinglePositiveOptionalInt32")
```

Comment:
Are these messages beneficial? 
- `swift test`'s output is already grouped by test case and test method,
- and you would always need to update them if you change the method name 🙂

Reply:
Agree 👍